cni/pkg/ebpf/server/options.go

```diff
@@ -0,0 +1,26 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+// RedirectArgs provides all of the configuration parameters for the redirection.
+type RedirectArgs struct {
```

Comment:
can you add comments what each one means

Reply:
This is really nit, but Go standard practice is to comment above the field, not beside. This supports godocs.